### DISCUSSION TITLE
colorMode and colorModeSwitch not working

### DISCUSSION THREAD

**Discussion by DaniBCD** [vuepress-reco/vuepress-theme-reco#289] [Q&A]
Hi, tried using colorMode as stated here https://vuepress-theme-reco.recoluan.com/en/docs/theme/appearance.html#introduction and colorModeSwitch from here https://ecosystem.vuejs.press/themes/default/config.html#colormodeswitch but it just doesn't work.

**Comment by recoluan** [comment]
colorModeSwitch was added in 2.0.0-rc.15
